### DISCUSSION TITLE
forge联机不了

### DISCUSSION THREAD

**Discussion by qianxiang789** [Meloong-Git/PCL#1995] [提问]
哥哥们，刚才和我朋友们准备整点mod然后联机，但是我们开了局域网联机后却并没有找到房间，但是我们换成原版的时候我们就可以联机了，这是为什么啊，求解答，谢谢

**Comment by WForst-Breeze** [comment]
提供一下相关截图谢谢~

**Reply by qianxiang789** [reply]
![Minecraft_ 1 19 4 8_6_2023 上午11_28_31](https://github.com/Hex-Dragon/PCL2/assets/135928729/45b41299-8933-45ba-8d93-06ee4156d3e3)
![Minecraft_ 1 19 4 8_6_2023 上午11_25_27](https://github.com/Hex-Dragon/PCL2/assets/135928729/76a4d605-60e0-4a50-9d85-a2fb67703fdc)
![Minecraft 1 19 4 8_6_2023 上午11_26_58](https://github.com/Hex-Dragon/PCL2/assets/135928729/1936ae8b-7478-42b0-ad22-d58d1e093a4b)
![Minecraft 1 19 4 8_6_2023 上午11_26_49](https://github.com/Hex-Dragon/PCL2/assets/135928729/bae4938e-2046-422b-bf15-dec3a00da339)

**Reply by qianxiang789** [reply]
就是用forge版本找不到局域网房间，但是普通版本就能找到

**Comment by Pigeon0v0** [comment]
如果加了自定义局域网之类的联机 Mod 麻烦去掉

**Reply by qianxiang789** [reply]
可是我刚才试了一下，把mod全部去掉，还是找不到房间

**Comment by qianxiang789** [comment]
我换了个低版本的就能找到了，可能是版本太高的问题，麻烦各位了

**Comment by WorldHim** [comment]
本地服务器有较大概率无法被搜索到，建议通过手动输入 IP / 域名连接

**Reply by qianxiang789** [reply]
试过了，还是不行，用低一点的版本是可以联，但高版本就不行了
